.github/workflows/ci.yml

```
@@ -33,7 +33,9 @@ jobs:
       run: |
         cargo check
         cargo test
-        cargo bench
+        cargo bench check
+        cargo fuzz check
+        cargo +nightly fuzz run fuzz_value 
```

```suggestion
        cargo +nightly fuzz run fuzz_value -- -max_total_time=5m
```

Otherwise the fuzzer will run indefinitely if no issues are found.